src/app/form/(formSteps)/review/page.tsx

```diff
@@ -1,9 +1,82 @@
 "use client";
 
-import React from "react";
+import { LoadingSpinner } from "@/app/form/(formSteps)/components/LoadingSpinner";
+import { getValue } from "@/app/form/_utils/sessionStorage";
+import FormProgressButtons from "@form/(formSteps)/components/FormProgressButtons";
+import { routeToNextStep, useFormProgressPosition } from "@form/_utils/formProgressRouting";
+import { Form } from "@trussworks/react-uswds";
+import { useRouter } from "next/navigation";
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
 
-const FormStep: React.FC = () => {
-  return <div>hi</div>;
+const ReviewStep: React.FC = () => {
+  const router = useRouter();
+  const formProgressPosition = useFormProgressPosition();
+  const { handleSubmit } = useForm<object>({
+    defaultValues: {},
+  });
+  const [loading, setLoading] = useState(true);
+  setTimeout(() => {
+    setLoading(false);
+  }, 2000);
+
+  return (
+    <div>
+      <LoadingSpinner loading={loading}>
+        <Form
+          onSubmit={handleSubmit(() => {
+            routeToNextStep(router, formProgressPosition);
+          })}
+          className="maxw-full"
+        >
+          <div className="maxw-tablet" lang="cs">
+            <h2 role="figure">Personal information</h2>
+            <label htmlFor="input" aria-checked>
+              First name
+            </label>
+            <div style={{ marginLeft: "40px" }}>{getValue("firstName")}</div>
+            <dt>Last name</dt>
+            <div style={{ marginLeft: "40px" }}>{getValue("lastName")}</div>
```

Comment:
Since we're using Description Term (`<dt>`) elements to label these pieces of data, we ought to use corresponding Description Details elements (`<dd>`)
```suggestion
            <dd style={{ marginLeft: "40px" }}>{getValue("lastName")}</dd>
```